src/Test/QuickCheck/Data/ApproxNumber.purs

```diff
@@ -1,13 +1,21 @@
-module Test.QuickCheck.Data.ApproxNumber where
+module Test.QuickCheck.Data.ApproxNumber
```

Comment:
Maybe now is a good time to remove the `Eq` and `Ord` instances here, since they are not valid.